api/src/main/java/me/lokka30/treasury/api/permission/NodeHolder.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/ArcanePlugins/Treasury>.
+ */
+
+package me.lokka30.treasury.api.permission;
+
+import java.util.Collection;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import me.lokka30.treasury.api.common.misc.TriState;
+import me.lokka30.treasury.api.common.response.Response;
+import org.jetbrains.annotations.NotNull;
+
+public interface NodeHolder {
+
+    @NotNull UUID getUniqueId();
+
+    @NotNull CompletableFuture<Response<Collection<Node<?>>>> allNodes();
```

Comment:
```suggestion
    @NotNull CompletableFuture<Response<Collection<Node<?>>>> getNodes();
```

Reply:
breaks consistency with `allNodesWithContext` and `allNodesWithType`